bluetooth-proxy/m5stack-atomS3.yaml

```
@@ -0,0 +1,43 @@
+substitutions:
+  name: atom-bluetooth-proxy
+  friendly_name: Bluetooth Proxy
+
+esphome:
+  name: ${name}
+  friendly_name: ${friendly_name}
+  name_add_mac_suffix: true
+  project:
+    name: esphome.bluetooth-proxy
+    version: "1.0"
+
+esp32:
+  board: m5stack-atoms3
+  variant: esp32s3
+  framework:
+    type: esp-idf
```

We really should use IDF 5+ for S3-based devices. It will support more boards and prevent boot loops.
```suggestion
    type: esp-idf
    version: 5.0.2
    platform_version: 6.3.2
```